ISLE/isleapp.cpp

```diff
@@ -287,9 +286,6 @@ SDL_AppResult SDL_AppInit(void** appstate, int argc, char** argv)
 
 SDL_AppResult SDL_AppIterate(void* appstate)
 {
-	if (g_closed) {
```

Comment:
Are we sure we don't need this?

Reply:
It is needed. I accidentally removed in a late refactor where I extracted debugapp from the IsleApp class into its own file.